ci/shell.nix

```diff
@@ -127,6 +127,9 @@ stdenvNoCC.mkDerivation ({
   # it's broken at the moment
   hardeningDisable = lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [ "stackprotector" ];
 
+  # Enabling rust-analyzer extension in VSCode
+  RUST_SRC_PATH = "${pkgs.rust.packages.stable.rustPlatform.rustLibSrc}";
```

Comment:
This potentially uses different rust version than is used for compilation, may or may not be a problem. Would `rustStable.rustPlatform.rustLibSrc` work here?

Reply:
Trying `RUST_SRC_PATH = "${rustStable.rustPlatform.rustLibSrc}";` gives 

```
error: while evaluating the attribute 'RUST_SRC_PATH' of the derivation 'trezor-firmware-env' at /nix/store/bn14fy2mc2p1q44681il8idfv5fr5x3z-source/pkgs/stdenv/generic/make-derivation.nix:205:7:
attribute 'rustPlatform' missing, at /home/jmusil/trezor-firmware/ci/shell.nix:132:22
```

Reply:
Ah sorry, can you try with https://github.com/trezor/trezor-firmware/commit/e0735b5645420585b54a64c5483906110f621579? Had to break out [nix repl](https://nixos.wiki/wiki/Nix_command/repl) to find the attribute:(